src/scripts/components/Input.jsx

```diff
@@ -40,36 +41,36 @@ class Input extends React.Component {
   _handleChange = (event) => {
     this.setState({ value: event.target.value });
 
-    if (this.props.onChange && typeof (this.props.onChange === 'function')) {
```

Comment:
Thanks for getting rid of these.. In this case it was doing absolutely nothing since it was always checking "typeof true/false", which always returns a truthy value. They should have been written:
```
typeof this.props.onChange === 'function'
```
which would return true or false..

vs
```
typeof (this.props.onChange === 'function')
```
will return 'boolean'.